stdlib/public/core/Integers.swift.gyb

```diff
@@ -223,7 +223,7 @@ def operatorComment(operator, fixedWidth):
   /// Returns the remainder of dividing the first value by the second.
   ///
   /// The result of the remainder operator (`%`) has the same sign as `lhs` and
-  /// is less than `abs(rhs)`.
+  /// is less than `rhs.magnitude`.
```

Comment:
This is still not quite right; it's the *magnitude* of the result that is less than the magnitude of `rhs`.